packages/@okta/vuepress-site/docs/reference/api/devices/index.md

```diff
@@ -612,17 +612,11 @@ curl -v -X GET \
 
 Lists all [Users](/docs/reference/api/users/#user-object) for a Device by `deviceId`
 
-#### Request parameters
+#### Request parameter
 
 | Parameter | Description                                                | ParamType | DataType | Required | Default |
 | --------- | ---------------------------------------------------------- | --------- | -------- | -------- | ------- |
-| after     | Specifies the pagination cursor for the next page of users | Query     | String   | FALSE    |         |
 | id        | ID of the Device                                          | URL       | String   | TRUE     |         |
```

Comment:
Do you want to change this parameter name to match the method description and the request example?
```suggestion
| deviceId        | ID of the Device                                          | URL       | String   | TRUE     |         |
```